fetch.bs

```diff
@@ -1093,27 +1097,35 @@ run these steps:
  are <a>ASCII digits</a>, from <var>data</var> given <var>position</var>.
 
  <li><p>If the <a>code point</a> at <var>position</var> within <var>data</var> is not U+002D (-),
- then return false.
+ then return failure.
 
  <li><p>Advance <var>position</var> by 1.
 
  <li><p>Let <var>rangeEnd</var> be the result of <a>collecting a sequence of code points</a> that
  are <a>ASCII digits</a>, from <var>data</var> given <var>position</var>.
 
- <li><p>If <var>position</var> is not past the end of <var>data</var>, then return false.
+ <li><p>If <var>position</var> is not past the end of <var>data</var>, then return failure.
 
  <li>
```

Comment:
If an `<li>` contains a single `<p>` there should not be a newline as is done above.